backend/FwLite/MiniLcm/SyncHelpers/WritingSystemSync.cs

```diff
@@ -0,0 +1,59 @@
+using MiniLcm;
+using MiniLcm.Models;
+using MiniLcm.SyncHelpers;
+using SystemTextJsonPatch;
+
+public static class WritingSystemSync
```

Comment:
this class should be in the `MiniLcm.SyncHelpers;` namespace

Reply:
Done in commit 237b32f1, and I also moved PartOfSpeechSync and SemanticDomainSync to that namespace since I noticed they were also not namespace.